plugins/jobs/server/models/job.py

```diff
@@ -467,3 +463,11 @@ def _computeFields(self, kwargs, includeLogDefault=False):
         if fields is None and not kwargs.pop('includeLog', includeLogDefault):
             fields = {'log': False}
         return fields
+
+    def getAllTypesAndStatuses(self, user=None):
```

Comment:
Hi @mgrauer, Thank you very much for all your review and comments. 
About this comment and the comment above, the whole story was like this. Before I implemented the admin jobs page, the jobs rest endpoint uses a string 'none' to indicate for anonymous jobs. Since I was just started, not wanting to break other's code. I created a dedicated listAll to list all jobs. 
Now, when I come back to this for the graph, as I am more familiar with the code, I found it redundant to have two sets of methods for list and list all. So, I removed the listall() and learned existing user='none' string and user = 'all' for filtering jobs. 
But I agree it is kind of confusing, I'll do some refactoring or adding comments to clear thing up.